web/src/components/Wrapper.js

```diff
@@ -1,41 +1,43 @@
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React, { useEffect } from 'react';
+import { withRouter } from 'react-router';
 import Header from './Header';
 import Footer from './Footer';
 
-class Wrapper extends Component {
-  componentDidMount() {
-    const { isDev } = this.props;
+const grayPaths = ['/create-account', '/edit-account', '/login', '/me'];
+
+const Wrapper = ({ children, isDev, location: { pathname } }) => {
+  useEffect(() => {
     if (isDev) {
-      this.addTota11y();
+      const script = document.createElement('script');
+      script.src = '/_dev/tota11y.min.js';
+      document.body.appendChild(script);
     }
-  }
+  }, []);
 
-  addTota11y = () => {
-    const script = document.createElement('script');
-    script.src = '/_dev/tota11y.min.js';
-    document.body.appendChild(script);
-  };
+  const isGray = grayPaths.indexOf(pathname) >= 0;
+  const showSiteTitle = pathname === '/';
 
-  render() {
-    const { children } = this.props;
-    const showSiteTitle = document.location.pathname === '/';
-    return (
-      <div className="site">
-        <a href="#start-main-content" className="skip-nav">
-          Skip to main content
-        </a>
-        <Header showSiteTitle={showSiteTitle} />
-        {children}
-        <Footer />
-      </div>
-    );
-  }
-}
+  return (
+    <div className={`site${isGray ? ' site--gray' : ''}`}>
+      <a href="#start-main-content" className="skip-nav">
+        Skip to main content
+      </a>
+      <Header showSiteTitle={showSiteTitle} />
+      {children}
+      <Footer />
+    </div>
+  );
+};
 
 Wrapper.propTypes = {
   children: PropTypes.node.isRequired,
-  isDev: PropTypes.bool.isRequired
+  isDev: PropTypes.bool.isRequired,
+  location: PropTypes.shape({
+    pathname: PropTypes.string.isRequired
+  }).isRequired
 };
 
-export default Wrapper;
+export default withRouter(Wrapper);
```

Comment:
Getting the location from the redux state means that the wrapper will re-render when the location changes, so the gray class will appear/disappear as appropriate. It also turns out this fixes a bug with the header not switching from the site title/dashboard link correctly.